### DISCUSSION TITLE
Is it possible to accept non trade confirmations using this?

### DISCUSSION THREAD

**Discussion by ustricavelma** [dyc3/steamguard-cli#235] [Q&A]
I was trying to accept my steam guard confirmations using your code to transfer my auth to mobile but im getting an error.
ERROR - Failed to get trade confirmations: Error("invalid type: null, expected a string", line: 1, column: 224)
I'm not rly good at programming but i assume this is just not a feature?

**Comment by dyc3** [comment]
Yes it should be possible. Can you post your logs with `-v debug` enabled?

**Reply by ustricavelma** [reply]
I'm not sure im doing it right but im getting this.
ERROR - Failed to get trade confirmations: Error("invalid type: null, expected a string", line: 1, column: 223)
INFO - failed to get trade confirmations, asking user to log in
INFO - Refreshing access token...
DEBUG - HTTP Request: POST https://api.steampowered.com/IAuthenticationService/GenerateAccessTokenForApp/v1
DEBUG - Response HTTP status: 200 OK
DEBUG - HTTP Header x-eresult: 1
INFO - Successfully refreshed access token, no need to prompt to log in.
DEBUG - HTTP Request: POST https://api.steampowered.com/ITwoFactorService/QueryTime/v1

**Reply by dyc3** [reply]
Turns out i forgot to log some stuff that would make this easier to debug. I'll get a new release out by the end of today so we can get some better logs for this.

**Reply by ustricavelma** [reply]
Brother, you are amazing!

**Comment by ustricavelma** [comment]
I've also tried to accept some trade confirmations. Worked fine with both trade and market

**Comment by dyc3** [comment]
Ok, I've published `v0.9.3`. Download it and try it again.

**Reply by ustricavelma** [reply]
Heyo, getting this now


INFO - Successfully refreshed access token, no need to prompt to log in.
DEBUG - HTTP Request: POST https://api.steampowered.com/ITwoFactorService/QueryTime/v1
DEBUG - Response HTTP status: 200 OK
DEBUG - HTTP Header x-eresult: 1
DEBUG - Confirmations response: {"success":true,"conf":[{"type":5,"type_name":"Account details","id":"13831364158","creator_id":"4486196268090979848","nonce":"11118003665658287603","creation_time":1687821676,"cancel":"Cancel","accept":"Confirm","icon":null,"multi":false,"headline":"Change phone number","summary":[""],"warn":null}]}
ERROR - Failed to get trade confirmations: Error {
    path: Path {
        segments: [
            Map {
                key: "conf",
            },
            Seq {
                index: 0,
            },
            Map {
                key: "icon",
            },
        ],
    },
    original: Error("invalid type: null, expected a string", line: 1, column: 224),
}
INFO - failed to get trade confirmations, asking user to log in
INFO - Refreshing access token...
DEBUG - HTTP Request: POST https://api.steampowered.com/IAuthenticationService/GenerateAccessTokenForApp/v1

**Reply by dyc3** [reply]
Perfect! I'll file a bug for it now and fix it soon. See #238

**Reply by ustricavelma** [reply]
Awesome! Thank you.

**Reply by ustricavelma** [reply]
It works! Thank you again for saving me from 14days of tradeban, you are amazing.
